node/pkg/processor/processor.go

```diff
@@ -474,8 +467,5 @@ func (p *Processor) vaaWriter(ctx context.Context) error {
 
 // GetFeatures returns the processor feature string that can be published in heartbeat messages.
 func GetFeatures() string {
-	if batchCutoverComplete() {
-		return "processor:batching"
```

Comment:
Is there a way that we can get rid of "p2p:new_gossip" from the guardian heartbeat messages?  Should that be an issue?

Reply:
I'm removing lines 531-533 in `p2p.go`. That will remove it.